### PR TITLE
ignore more files and add CoC to doc

### PR DESCRIPTION
**Pull request by evgeni**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by evgeni** [comment]
@ekohl agreed

File: gem2rpm/foreman_plugin.spec.erb [hunk 1]
```diff
@@ -27,9 +27,13 @@ precompile_flags << ' -a' if has_apidoc
 precompile_flags << ' -s' if has_assets || has_webpack
 
 config.rules[:doc] << /\/?CHANGES/
+config.rules[:doc] << /CODE_OF_CONDUCT(\.md)?/
```
Comment (mmoll) [inline gem2rpm/foreman_plugin.spec.erb]: which gems bring such a file?

Comment (ekohl) [reply gem2rpm/foreman_plugin.spec.erb]:
```
$ rg -l CODE_OF_CONDUCT packages
packages/plugins/rubygem-fog-proxmox/rubygem-fog-proxmox.spec
packages/foreman/rubygem-googleauth/rubygem-googleauth.spec
packages/foreman/rubygem-patternfly-sass/rubygem-patternfly-sass.spec
packages/plugins/nodejs-react-json-tree/nodejs-react-json-tree.spec
packages/foreman/rubygem-google-api-client/rubygem-google-api-client.spec
packages/foreman/rubygem-secure_headers/rubygem-secure_headers.spec
packages/foreman/rubygem-fog-rackspace/rubygem-fog-rackspace.spec
packages/foreman/rubygem-fog-openstack/rubygem-fog-openstack.spec
```

Comment (mmoll) [reply gem2rpm/foreman_plugin.spec.erb]: Ah, so none of the plugins maintained in the theforeman org, all good. :+1:

Comment (ekohl) [reply gem2rpm/foreman_plugin.spec.erb]: Yes, probably why it makes more sense to add it to `scl.spec.erb` and gem2rpm itself.

**Pull request merged**